.github/images/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM quay.io/centos/centos:stream8 AS base
+
+ENV PGHOST=postgres
+ENV PGUSER=foreman
+ENV PGPASS=foreman
+ENV RAILS_ENV=test
+ENV GIT_COMMITTER_NAME="gh_actions"
+ENV GIT_COMMITTER_EMAIL="gh_actions@foreman_virt_who_configure.foreman"
+ENV WORKDIR=/projects/foreman
+ENV BUNDLE_PATH=vendor/bundle
+
+RUN \
+  dnf module install -y ruby:2.7 nodejs:14 postgresql:12;\
```

Comment:
I think we can drop nodejs:14 but curious if it will fail on other things, if it does we can keep it, if it does not fail, it will make the container spinup faster